packages/components/organisms/f-header/src/components/_tests/Navigation.test.js

```diff
@@ -639,7 +652,7 @@ describe('Navigation', () => {
             expect(wrapper.find('[data-test-id="login"]').exists()).toBe(false);
         });
 
-        it('should NOT show "navLinks" if `showLoginInfo: false` and the user is logged in and has nav link data', async () => {
+        it('should NOT show "navLinks" if `: false` and the user is logged in and has nav link data', async () => {
```

Comment:
Deleting `showLoginInfo` from the description makes this a little unclear as to what needs to be false. Is there a specific variable that when it's false means the `navLinks` don't show now?